guides/doc-Release_Notes/topics/katello-4.14.1.adoc

```diff
@@ -0,0 +1,45 @@
+= Katello 4.14.1
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1878[Redmine]
+
+== Katello
+
+* pass:[Container push sometimes makes duplicate repos due to race condition] - https://projects.theforeman.org/issues/37785[#37785]
+* pass:[Deleting published repos from product page doesn't work right] - https://projects.theforeman.org/issues/37782[#37782]
+* pass:[Handle empty CVE in InfoProvider content_view_info] - https://projects.theforeman.org/issues/37779[#37779]
+* pass:[Use :default_location_subscribed_hosts in registration] - https://projects.theforeman.org/issues/37703[#37703]
+* pass:[Pagination within Packages wizard is wonky] - https://projects.theforeman.org/issues/37587[#37587]
+
+=== Content Credentials
```

Comment:
```suggestion
=== Content credentials
```
Unless the headings are generated, we use sentence case. Please also correct for other headings.

Reply:
@Lennonka they are generated but fixed them anyways, will raise an issue against the redmine script. Let me know how this looks :)